new-lamassu-admin/src/pages/Maintenance/CashCassettesFooter.js

```diff
@@ -60,7 +61,8 @@ const CashCassettesFooter = ({
           <TxOutIcon className={classes.icon} />
           <Info2 className={classes.iconLabel}>Cash-out:</Info2>
           <Info1 className={classes.valueDisplay}>
-            {totalInCassettes} {currencyCode}
+            {isValidNumber(totalInCassettes) ? totalInCassettes : 0}{' '}
```

Comment:
I don't think this fixes it. What happens when there's a mix of cash-out and non cash-out machines? Probably better to validate on the reducer